jxl_cli/src/enc/numpy.rs

```diff
@@ -53,7 +53,6 @@ fn numpy_bytes(image_data: ImageData<f32>, num_channels: usize) -> Vec<u8> {
     let (width, height) = size;
 
     for frame in image_data.frames {
-        assert_eq!(frame.size, size);
```

Comment:
I am not convinced we should remove this assert...

Reply:
Done